packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -203,7 +203,7 @@ export class ScmTreeWidget extends TreeWidget {
 
                 if (widget) {
                     const diffNavigator = this.diffNavigatorProvider(widget.editor);
-                    if (diffNavigator.canNavigate() && diffNavigator.hasPrevious()) {
+                    if (diffNavigator.canNavigate() /* &&  diffNavigator.hasPrevious() */) {
```

Comment:
These changes are not obvious. Why are they needed?

Reply:
Removed early on, then fixed later, but I missed this line, and one `hasNext`. Good catch.